scripts/Tools/Makefile

```diff
@@ -302,8 +296,13 @@ endif
 # Force PIO to use MPI_Isends instead of the default, MPI_Irsends
 ifeq ($(PIO_VERSION),2)
   EXTRA_PIO_CPPDEFS = -DUSE_MPI_ISEND_FOR_FC
+  ifneq ($(COMPILER),nag)
+    # Needed by SCORPIO not default PIO, does not work with NAG compiler
+    SUPPORTS_CXX := TRUE
```

Comment:
Should this be USE_CXX (instead of SUPPORTS_CXX)?

Reply:
yes, sorry - I'll push a fix.

Reply:
@jayeshkrishna fixed now.